test/fixtures/definitions/with-invalid-checksum

```diff
@@ -1 +1 @@
-install_package "package-1.0.0" "http://example.com/packages/package-1.0.0.tar.gz#invalid" copy
+install_package "package-1.0.0" "http://example.com/packages/package-1.0.0.tar.gz#invalidbb4250dee0b9dd3d4d722f9c64b2bacfc805d1b6eba7426bda72dd3c5" copy
```

Comment:
Can the invalid checksum be spaced with zeroes instead so that it's clear that it's not an actual checksum? Will be nicer to read I think.